support/crown/src/main.rs

```diff
@@ -64,7 +65,16 @@ fn main() -> ExitCode {
     let handler =
         rustc_session::EarlyDiagCtxt::new(rustc_session::config::ErrorOutputType::default());
     rustc_driver::init_logger(&handler, rustc_log::LoggerConfig::from_env("CROWN_LOG"));
-    let args: Vec<_> = std::env::args().collect();
+    let mut args: Vec<_> = std::env::args().collect();
+
+    // Setting RUSTC_WRAPPER causes Cargo to pass 'rustc' as the first argument.
```

Comment:
Do we use `RUSTC_WRAPPER` or `CARGO_BUILD_RUSTC` or a mixture of both now? Wouldn't just one of them be sufficient?

Reply:
One is sufficient, but my muscle memory uses RUSTC_WRAPPER when running manually.